src/Components/Notifications/NoticeBoard.tsx

```diff
@@ -30,49 +30,65 @@ export const NoticeBoard: any = () => {
   }, [dispatch]);
 
   let notices;
+
   if (data && data.length) {
     notices = (
       <div className="grid grid-cols-1 sm:grid-cols-2 xl:grid-cols-3 gap-x-6 gap-y-6">
         {data.map((item) => (
-          <Card key={`usr_${item.id}`} className="rounded w-full shadow">
-            <CardContent>
-              <div className="bg-white">
-                <div className="text-justify text-lg">{item.message}</div>
-                <div className="text-gray-700 text-md mt-2">
-                  {`${item.caused_by.first_name} ${item.caused_by.last_name}`} -{" "}
-                  <span className="text-primary-700 font-bold">
-                    {item.caused_by.user_type}
-                  </span>
-                </div>
-                <div className="text-xs text-gray-900">
-                  {t("on")}: {formatDate(item.created_date)}
-                </div>
+          <div
+            key={`usr_${item.id}`}
+            className="max-w-sm rounded overflow-hidden shadow-lg "
+          >
+            <div className="px-6 py-4">
+              <div className="text-justify text-lg">{item.message}</div>
+              <div className="text-gray-700 text-md mb-2 mt-2">
+                {`${item.caused_by.first_name} ${item.caused_by.last_name}`} -{" "}
+                <span className="text-primary-700 font-bold">
+                  {item.caused_by.user_type}
+                </span>
+              </div>
+              <div className="text-gray-900 text-xs">
+                {t("on")}: {formatDate(item.created_date)}
               </div>
-            </CardContent>
-          </Card>
+            </div>
+          </div>
         ))}
       </div>
     );
   } else {
     notices = (
-      <Card key="no-notice" className="my-4 rounded-lg">
-        <CardContent>
-          <div className="text-xl text-center semibold">
-            {t("no_notices_for_you")}
+      <div className=" m-auto max-w-sm rounded overflow-hidden shadow-lg ">
+        <img
+          className="w-full"
+          src={"/images/Notificationbell.png"}
+          alt="bell"
+        ></img>
+        <div className="px-6 py-4">
+          <div className="font-bold text-xl mb-2">
+            You Dont Have Any Notices
           </div>
-        </CardContent>
-      </Card>
+          <p className="text-gray-700 text-base">
+            Did you know: We could render a fun fact about health here
+          </p>
+        </div>
+
+        <div className="px-6 pt-4 pb-2">
+          <ButtonV2 className=" mx-28 " href="/">
+            Go Home
+          </ButtonV2>
+        </div>
+      </div>
     );
   }
 
   if (isLoading) return <Loading />;
   return (
     <div className="px-6">
-      <PageTitle
+      <Page
         title={t("Notice Board")}
         hideBack={true}
         breadcrumbs={false}
-      />
+      ></Page>
       <div>{notices}</div>
     </div>
```

Comment:
1. Page component shouldn't be wrapped with a parent padding. The goal is to have consistent layout and padding across the platform by using the new generic Page component.
2. Page component should wrap all the contents. Why is the notices not wrapped?